data/json/furniture.json

```diff
@@ -22,6 +23,7 @@
         "id" : "f_rubble",
         "name": "pile of rubble",
         "symbol": "^",
+        "description": "Pile of various metals, bricks, and other building materials. Nothing salvagable.",
```

Comment:
There should be two spaces between sentences, like this: "First sentence is here.  Second sentence is here."
Also, the description should note that it can be removed with a shovel.